llama-index-integrations/llms/llama-index-llms-nvidia-ai-playground/llama_index/llms/nvidia_ai_playground/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict, Sequence, List
+
+from llama_index.core.base.llms.types import ChatMessage
+
+from openai.types.chat import ChatCompletionMessageParam
+from openai.types.chat.chat_completion_message import ChatCompletionMessage
+
+AI_PLAYGROUND_MODELS: Dict[str, int] = {
+    "playground_llama2_13b": 4096,
+    "playground_llama2_code_13b": 100_000,
+    "playground_llama2_70b": 4096,
+    "playground_mistral_7b": 8192,
+    "playground_nv_llama2_rlhf_70b": 4096,
+    "playground_yi_34b": 200_000,
+    "playground_nemotron_steerlm_8b": 3072,
+    "playground_mixtral_8x7b": 32_000,
+    "playground_llama2_code_34b": 100_000,
+    "playground_steerlm_llama_70b": 3072,
+}
```

Comment:
models to add -
- [ ] mistralai/mistral-7b-instruct-v0.2
- [ ] mistralai/mixtral-8x7b-instruct-v0.1
- [ ] google/gemma-7b
- [ ] meta/codellama-70b
- [ ] meta/llama2-70b
- [ ] cohere/aya-101
- [ ] cohere/command-r

Reply:
Added!

Reply:
@chrisalexiuk-nvidia the cohere ones aren't actually part of api catalog, let's remove them

Reply:
Sounds good!